.github/workflows/addons-pr-validator.yml

```diff
@@ -0,0 +1,45 @@
+name: 3rd Party Addons PR Validator
+on:
+  workflow_dispatch:
+  push:
+    # TODO: Re-enable this, to prevent running when not necessary
+    # paths:
+    #   - 'assets/addon_browser/'
+    #   - '.github/workflows/addons-pr-validator.yml'
+  pull_request:
+    # TODO: Re-enable this, to prevent running when not necessary
+    # paths:
+    #   - 'assets/addon_browser/'
```

Comment:
What about those?

Reply:
Ahh yeah I probably should try activating these again..
They didn't work when I initially tried it, so I kept them disabled for the time being.
My idea was to do it later, when everything is merged and working. If you'd like, I can try and get them to work again, before you merge this?